surf.deckr.deckr.yaml

```diff
@@ -0,0 +1,201 @@
+app-id: surf.deckr.deckr
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+command: deckr
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+
+modules:
+  - name: deckr
+    buildsystem: simple
+    build-commands:
+      # - mkdir -p deckr
+      # - tar -xf Deckr-Linux-Portable.tar.gz -C deckr
+      - mv deckr /app/deckr
+
+      - chmod +x /app/deckr/deckr
+      - mkdir -p /app/bin
+      - ln -s /app/deckr/deckr /app/bin/deckr
+
+      - mv gpt /app/gpt
+
+      - install -Dm644 surf.deckr.deckr.metainfo.xml -t /app/share/metainfo/
+      - install -Dm644 app_icon_32.png /app/share/icons/hicolor/32x32/apps/surf.deckr.deckr.png
+      - install -Dm644 app_icon_64.png /app/share/icons/hicolor/64x64/apps/surf.deckr.deckr.png
+      - install -Dm644 app_icon_128.png /app/share/icons/hicolor/128x128/apps/surf.deckr.deckr.png
+      - install -Dm644 app_icon_256.png /app/share/icons/hicolor/256x256/apps/surf.deckr.deckr.png
+      - install -Dm644 app_icon_512.png /app/share/icons/hicolor/512x512/apps/surf.deckr.deckr.png
+      - install -Dm644 surf.deckr.deckr.desktop -t /app/share/applications/
+    sources:
+      - type: archive
+        url: https://d3sp89svrxzy0t.cloudfront.net/dashboard/Deckr-Linux-Portable.tar.gz
+        # url: https://d3sp89svrxzy0t.cloudfront.net/dashboard/Deckr-Debug.tar.gz
+        sha256: 1c2b41d76ee068a3b0fc3699f347c52ce210efd91ba560ae9248e9aee70fd684
+        dest: deckr
+
+      - type: archive
+        url: https://d3sp89svrxzy0t.cloudfront.net/dashboard/gpt-libs.tar.gz
```

Comment:
These URLs are not versioned? This will break the build if shasum changes.

Reply:
Also why was this changed to cloudfront.net download link?

Reply:
It was get.cocoatech.com and you complained, so this is what get.cocoatech.com points to, my S3 cloud front account.   same location.

I've never versioned it.  It's too much trouble.  And it's worked fine so far.  Maybe one day I'll do it, but I don't think it's that important.

What is the domain question?  I want this to be verified.  The official download site is https://deckr.surf, but the old id was com.cocoatech.deckr, so I had t change it to match the official web page where I will put that well-known file.

Any thing else?

Reply:
I didn't complain I wanted some verification that you own the two domains here. Can you please do that? https://github.com/flathub/flathub/pull/5529#discussion_r1730588024

Reply:
i answered it